src/config/config.ts

```diff
@@ -300,7 +300,7 @@ export function initializeLoadConfiguration() {
 
     const configServiceUrl =
         getBrowserWindow().frontendConfig.configurationServiceUrl ||
-        `${APIROOT}index`;
```

Comment:
@inodb @haynescd this looks like maybe a typo right?  it DID need to be changed but not sure why it was changed to "index".